boilerplate/openshift/golang-osd-operator-osde2e/e2e-harness-generate.sh

```diff
@@ -71,9 +73,12 @@ const (
 // Test entrypoint. osde2e runs this as a test suite on test pod.
 func Test$REPLACE_FUNC(t *testing.T) {
 	RegisterFailHandler(Fail)
-
 	suiteConfig, reporterConfig := GinkgoConfiguration()
 	reporterConfig.JUnitReport = filepath.Join(testResultsDirectory, jUnitOutputFilename)
+
+ 	if _, ok := os.LookupEnv("DISABLE_TEST_RUN_RESULTS"); !ok {
```

Comment:
Can we call this `DISABLE_JUNIT_REPORT`?

Reply:
Yes, done!